cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -624,10 +625,51 @@ update msg model =
                     )
 
                 ShowSaveTestError ->
-                    ({model | showSaveTestError = True}, Cmd.none)
+                    ( { model | showSaveTestError = True }, Cmd.none )
 
         ServerGetAttributeResponse attributes ->
-            ( { model | customAttributes = attributes }
+            let
+                -- We also want set the attributeStates once the attributes are fetched
+                attributeEditorState : SDKDict.Dict AttrValueDetail ValueEditor.EditorState
+                attributeEditorState =
+                    attributes
+                        |> Dict.toList
+                        |> List.map
+                            (\( customAttributeId, customAttributeDetail ) ->
+                                let
+                                    selectedModuleName =
+                                        model.homeState.selectedModule
+                                            |> Maybe.map Tuple.second
+                                            |> Maybe.withDefault []
+
+                                    nodeId =
+                                        ModuleID selectedModuleName
+
+                                    attributeValueDetails =
+                                        AttrValueDetail customAttributeId nodeId
```

Comment:
Simple declarations like this make the code less readable in my opinion. There's no logic here and it seems to be used only once so inlining would make it read much better.

Reply:
This has been cleaned up